specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.1-preview.2/FormRecognizer.json

```diff
@@ -1638,6 +1638,10 @@
         "confidence": {
           "description": "Confidence value.",
           "$ref": "#/definitions/Confidence"
+        },
+        "appearance": {
+          "description": "Style and its confidence value.",
```

Comment:
"Text appearance properties."  We may add more properties in the future.

Reply:
Fixed.